statsmodels/doe/tests/test_lhs.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import numpy as np
+import numpy.testing as npt
```

Comment:
Importing the assert functions directly is more consistent with the other statsmodels code. It's also shorter given that they are heavily used in test files, i.e.
from numpy testing import assert_allclose, assert_almost_equal, ...